src/atopile/errors.py

```diff
@@ -359,28 +256,24 @@ def muffle_fatalities():
         )
         do_exit = True
 
-    except Exception as ex:
+    except* Exception as ex:
```

Comment:
This re-casts the underlying exception as a new ExceptionGroup, so the debugger only sinks to here
We should ideally not use this * syntax here